js/actions.js

```diff
@@ -0,0 +1,89 @@
+var startBtn = $('#startBtn');
```

Comment:
Wrap everything in a function to create a new scope for your code. You can use an immediately invoked function !function (){}() and put your stuff inside